js/components/RightsTips.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import PropTypes from 'prop-types';
+import { textStyles, colors } from '../styles';
+
+export default function Tips({ tips }) {
+  const tipsList = tips.map((tip) => {
+    return (
+      <View style={styles.container} key={tip}>
+        <Text style={textStyles.h3}>{tip}</Text>
+      </View>
+    );
+  });
+
+  return (
+    <View style={{ paddingTop: 50 }}>
+      <Text style={textStyles.h2}>{'Tips'}</Text>
+      <View>{tipsList}</View>
+    </View>
+  );
+}
+
+Tips.propTypes = {
+  tips: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Comment:
```suggestion
  tips: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
```
to avoid `[undefined, "foo", undefined]` being valid

Reply:
👍